desktop-app/src/store/index.ts

```diff
@@ -104,7 +104,7 @@ const schema = {
             type: {
               type: 'string',
             },
-            dpi: {
```

Comment:
We need to add a migration for this rename as well, to handle the config files on existing installations. 

Migrations exist in this file: https://github.com/responsively-org/responsively-app/blob/main/desktop-app/src/store/migrations.ts

Reply:
```typescript
'1.2.2': (store: Store) => {
    // Migrate dpi to dpr in custom devices
    try {
      const previousCustomDevices: any[] = store.get(
        'deviceManager.customDevices'
      ) as any[];
      const newCustomDevices: Device[] = previousCustomDevices.map((device) => {
        const newDevice = {
          ...device,
          dpr: device.dpi !== undefined ? device.dpi : device.dpr,
        };
        delete newDevice.dpi;
        return newDevice as Device;
      });
      store.set('deviceManager.customDevices', newCustomDevices);
      console.log(
        'Migration for 1.2.2 successful',
        store.get('deviceManager.customDevices')
      );
    } catch (e) {
      console.log('Migration for 1.2.2 failed', e);
    }
  },
```

Is this a valid approach for migration following the flow of existing code?

Reply:
Yes, looking good. Just make sure the version number is `1.13.0` as that will be the next release version.